### ISSUE TITLE
 Psycopg2 Warning

### ISSUE THREAD

**Issue by henrykironde** [weecology/retriever#1227]
Psycopg2 Warning:
```python
 psycopg2/__init__.py:144: UserWarning: The psycopg2 
wheel package will be renamed from release 2.8; in order to keep installing from 
binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
    """)
```
This will disappear with the upcoming release https://github.com/psycopg/psycopg2/milestone/14

**Comment by piyushchauhan** [comment]
I suppose work on this will begin as soon as v2.8 of psycopg2 will be released. Right? 
Beta Version has been released.

**Comment by zhangcandrew** [comment]
Fixed in #1301
